golang/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM dockerfile/ubuntu
+
+MAINTAINER Filip Gospodinov "filip@monetas.net"
+# use bash for docker RUN commands; we want associative arrays
+RUN rm /bin/sh && ln -s /bin/bash /bin/sh
+
+# use custom GOPATH such that it does not conlfict with GOPATHs that
+# live inside the host's $HOME which may be bind mounted into docker
+ENV GOPATH $HOME/go_monetas
+ENV GOROOT $HOME/go_dist/go
```

Comment:
If we mount the host's $HOME as the container's $HOME as well, we would not start with a fresh GOPATH on subsequent runs. Maybe we should use something under /opt as the GOPATH/GOROOT instead?